app/moves/index.js

```diff
@@ -23,9 +21,7 @@ const uuidRegex =
 
 // Define param middleware
 router.param('locationId', setFromLocation)
-router.param('date', setMoveDate)
```

Comment:
You could replace this with `setDateRange` now.

It's more efficient using it with [`router.param`](https://expressjs.com/en/api.html#router.param) as I believe express will only load that method in once, rather than each time for each route like we currently have.